services/hsts/hsts.service.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+const label = 'hsts preloaded'
+const { BaseJsonService } = require('..')
+
+const Joi = require('joi')
+const schema = Joi.object({
+  status: Joi.string().required(),
+}).required()
+
+module.exports = class HSTS extends BaseJsonService {
+  static get category() {
+    return 'monitoring'
+  }
+
+  static get route() {
+    return {
+      base: 'hsts',
+      pattern: ':domain',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Chromium HSTS preload',
+        namedParams: { domain: 'github.com' },
+        staticPreview: this.render({ status: 'preloaded' }),
+        keywords: ['security'],
+      },
+    ]
+  }
+
+  async fetch({ domain }) {
+    return this._requestJson({
+      schema,
+      url: `https://hstspreload.org/api/v2/status?domain=${domain}`,
```

Comment:
Could you use `options: { qs: { domain } }` here instead? That ensures correct query string escaping regardless of what users might send.

Reply:
Sure. I didn't know that was available, would look neater.